build/configs/artik053/audio/Make.defs

```diff
@@ -155,3 +155,7 @@ define DOWNLOAD
   @$(TOPDIR)/../build/configs/artik05x/artik05x_download.sh --board=artik053 $(1) $(2) $(3) $(4) $(5) $(6)
 endef
 
+define OTA_IMG
+  $(Q) $(TOPDIR)/../build/configs/artik05x/artik05x_ota.sh \
+    --board=artik053 --secure ota
```

Comment:
1. Is ```--secure``` needed to normal model?
2. Let's make it a line. We don't need to change a line.